khal/aux.py

```diff
@@ -112,6 +112,8 @@ def calc_day(dayname):
         return today
     if dayname == 'tomorrow':
         return today + timedelta(days=1)
+    if dayname == 'now':
```

Comment:
this function is called `calc_day` and currently only returns `datetime.date` objects, I don't think we should start returning `datetime.datetime` objects from here.